tests/test__collection_api.py

```diff
@@ -5601,6 +5601,77 @@ def test__aggregate_filter_wrong_options(self):
                 self.db.collection.aggregate(
                     [{'$project': {'filtered_items': {'$filter': option}}}])
 
+    def test__aggregate_map(self):
```

Comment:
Please add an equivalent test in test__mongomock

Reply:
Done.